### ISSUE TITLE
Nullpointer Exception

### ISSUE THREAD

**Issue by ChristianLuessenhop** [Refinitiv/Real-Time-SDK#146]
In the method **channelInformation** of class **com.thomsonreuters.ema.access.OmmConsumerImpl** NullpointerExceptions can occur.
And it does....

reactorChannel.channel() is null sometimes.
It is checked, but only componentInfo is set.

```

			if (reactorChannel.channel() == null ) {
				channelInformation.componentInfo("unavailable");
			}

```

Later it will be used nevertheless:

	
		
```
channelInformation.channelState(reactorChannel.channel().state());
			if (reactorChannel.channel() != null) {
				channelInformation.connectionType(reactorChannel.channel().connectionType());
				channelInformation.protocolType(reactorChannel.channel().protocolType());
				channelInformation.majorVersion(reactorChannel.channel().majorVersion());
				channelInformation.minorVersion(reactorChannel.channel().minorVersion());
				channelInformation.pingTimeout(reactorChannel.channel().pingTimeout());
			}
			else {
				channelInformation.connectionType(-1);
				channelInformation.protocolType(-1);
				channelInformation.majorVersion(0);
				channelInformation.minorVersion(0);
				channelInformation.pingTimeout(0);
			}
```

		
reactorChannel.channel().state() will throw NullpointerException

**Comment by APaias** [comment]
Hi @ChristianLuessenhop,
Thank you for reporting this issue.
An internal ticket has been created to track this issue. We're actively working to ensure that it is addressed.

**Comment by vlevendel** [comment]
Fix is included with tag, Real-Time-SDK-2.0.1.L1.
